### PR TITLE
Userinfo claim and id token push claim fixes

### LINKED ISSUE #304: Need to Update Openid connect plugin as userinfo reponse updated in oxd
old response
```
{
        "claims":{
            "sub": ["248289761001"],
            "name": ["Jane Doe"],
            "given_name": ["Jane"],
            "family_name": ["Doe"],
            "preferred_username": ["j.doe"],
            "email": ["janedoe@example.com"],
            "picture": ["http://example.com/janedoe/me.jpg"]
        }
    }
```

New response:
```
{
            "sub": "248289761001",
            "name": "Jane Doe",
            "given_name": "Jane",
            "family_name": "Doe",
            "preferred_username": "j.doe",
            "email": "janedoe@example.com",
            "picture": "http://example.com/janedoe/me.jpg"
}
```

### LINKED ISSUE #308: Handle intenal and external Post logout uri support
Kong should handle both internal(Kong proxy endpoint) and external(like CDN) URLs for post logout in openid connect plugin.

### PR DESCRIPTION
**Pull request by ldeveloperl1985**
close #304 
close #308

### CODE REVIEW

File: kong/plugins/gluu-openid-connect/access.lua [hunk 1]
```diff
@@ -20,11 +20,18 @@ local function process_logout(conf, session)
     -- TODO get rid of self parameter of get_protection_token(), make it as separate commit
     local ptoken = kong_auth_pep_common.get_protection_token(nil, conf)
 
+    local post_logout_redirect_uri
+    if conf.post_logout_redirect_path_or_url:sub(0, 1) == "/" then
```
Comment (altexy) [inline kong/plugins/gluu-openid-connect/access.lua]: In Lua index start with 1, should be sub(1,1)

Comment (ldeveloperl1985) [reply kong/plugins/gluu-openid-connect/access.lua]: Okay

File: kong/common/kong-common.lua [hunk 2]
```diff
@@ -353,8 +353,7 @@ _M.access_pep_handler = function(self, conf, hooks)
             kong.log.debug("no token, protected path")
             return hooks.no_token_protected_path(self, conf, protected_path, method)
         end
-        -- TODO shall we allow access if not conf.deny_by_default ?!
-        return kong.response.exit(403, { message = "Invalid request, no token and no protected path" })
+        return -- access allow, conf.deny_by_default = false
```
Comment (altexy) [inline kong/common/kong-common.lua]: Hm, where is the check for `conf.deny_by_default`?

Comment (ldeveloperl1985) [reply kong/common/kong-common.lua]: Hmm we missed this. Working on it

**Pull request merged**